astropy/io/fits/tests/test_diff.py

```diff
@@ -217,13 +261,37 @@ def test_trivial_identical_images(self):
         assert diff.identical
         assert diff.diff_total == 0
 
-    def test_identical_within_tolerance(self):
+    def test_identical_within_relative_tolerance(self):
         ia = np.ones((10, 10)) - 0.00001
         ib = np.ones((10, 10)) - 0.00002
-        diff = ImageDataDiff(ia, ib, tolerance=1.0e-4)
+        diff = ImageDataDiff(ia, ib, rtol=1.0e-4)
         assert diff.identical
         assert diff.diff_total == 0
 
+    def test_identical_within_absolute_tolerance(self):
+        ia = np.zeros((10, 10)) - 0.00001
+        ib = np.zeros((10, 10)) - 0.00002
+        diff = ImageDataDiff(ia, ib, rtol=1.0e-4)
+        assert not diff.identical
+        assert diff.diff_total == 100
+        diff = ImageDataDiff(ia, ib, atol=1.0e-4)
+        assert diff.identical
+        assert diff.diff_total == 0
+
+    def test_not_identical_within_rtol_and_atol(self):
```

Comment:
The function name implies that you are testing for "not identical" but then the actual test does the opposite. This is very confusing. Same comment for the test that follows below.